Poisson_surface_reconstruction_3/include/CGAL/Poisson_mesh_domain_3.h

```diff
@@ -0,0 +1,399 @@
+// Copyright (c) 2025 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Sven Oesau
+//
+//******************************************************************************
+// File Description :
+// class Poisson_mesh_domain_3. See class description.
+//******************************************************************************
+
+#ifndef CGAL_POISSON_MESH_DOMAIN_3_H
+#define CGAL_POISSON_MESH_DOMAIN_3_H
+
+#include <CGAL/license/Mesh_3.h>
+
+#include <CGAL/Labeled_mesh_domain_3.h>
+#include <CGAL/Poisson_reconstruction_function.h>
+
+namespace CGAL {
+
+/*!
+\ingroup PkgMesh3Domains
```

Comment:
`PkgMesh3Domains` does not exist probably missing `Mesh_3` tag file in settings file

Reply:
I forgot to change the labels after moving the header from one package to another.